### DISCUSSION TITLE
NMReDATA parsing

### DISCUSSION THREAD

**Discussion by andrecastro0o** [rdkit/rdkit#3868] [Development]
Dear developers,
At the [NFDI4Chem – Chemistry Consortium](https://nfdi4chem.de/) we (myself, @okoepler, @StroemPhi and @hujo91) are currently working with the NMReDATA (SDF based) format (see https://doi.org/10.1002/mrc.4737 for article on the format and [example files](https://github.com/NMReDATAInitiative/Examples-of-NMR-records) ), with the goal of creating a semantic knowledge graph from the information contained in it the NMReDATA file.

I have written a [simple parsing prototype](https://github.com/NFDI4Chem/NMReDATA_parsing-/blob/main/notebooks/Parse%20SDF.ipynb), using `rdkit.Chem.SDMolSupplier` and the parsing seems to work fine, but the data contained within the `NMREDATA_1D_Isotope` & `NMREDATA_2D_Isotope` such as Larmor, Pulseprogram, Spectrum_Location are not parsed, the returned value is a long string with this unparsed info inside. You can see this in the prototype mentioned above.

Long story short...My question would be if RDKIT development team would be interested in enhancing its code base to allow the full parsing of NMReDATA. 

Side note: I was not able to test yet, if the following tags are parsed correctly as we need to find/create an example which contains it 
* NMREDATA_ID
* NMREDATA_TEMPERATURE
* NMREDATA_CONCENTRATION (optional)
* NMREDATA_PH (optional)
* NMREDATA_FORMULA
* NMREDATA_INCHI
* NMREDATA_ALATIS

Looking forward to hear from you.
